metals/src/main/scala/scala/meta/internal/metals/codeactions/InsertInferredType.scala

```diff
@@ -67,11 +78,47 @@ class InsertInferredType(trees: Trees) extends CodeAction {
       title <- inferTypeTitle(name)
     } yield insertInferTypeAction(title)
 
-    actions.toList
+    def adjustTypeTree(tree: Defn): Option[l.Range] = tree match {
+      case Defn.Def(_, name, _, _, tpe, _) if tpe.isDefined =>
+        Some(name.pos.toLSP)
```

Comment:
If there are parameters it should point ad the last `)`. 
Otherwise the action removes parameters,

Reply:
Not sure what you mean here, do you maybe have a code example that this breaks on?

Reply:
this one fails if you add it
```scala
  checkEdit(
    "wrong-def-params".only,
    """|object A{
       |  def <<alpha>>(a: Int, b: String): String = 123
       |}""".stripMargin,
    """|object A{
       |  def alpha(a: Int, b: String): Int = 123
       |}""".stripMargin,
  )
```

Reply:
Thanks! I totally forgot about this edge case. It's fixed now.